src/app/Plans/components/PlanDetails.tsx

```diff
@@ -192,24 +201,41 @@ const PlanDetails: React.FunctionComponent<IPlanDetailsProps> = ({ plan }: IPlan
         <GridItem md={9} id="review-migration-type" aria-labelledby="migration-type-label">
           {plan.spec.warm ? 'warm' : 'cold'}
         </GridItem>
-        <GridItem md={3} id="migration-type-label">
-          Hooks
-        </GridItem>
-        <GridItem md={9} id="review-plan-hooks" aria-labelledby="migration-hooks-label">
-          <div>
-            <Grid>
-              <GridItem span={5} id="migration-plan-hooks-definition-label">
-                <label className="pf-c-form__label">Definition</label>
-                {plan.spec.vms[0].hooks?.map((hook) => hook.hook)}
+        {
+          // TODO: Add logic to handle dedicated Hook by VM
+          plan.spec.vms[0].hooks ? (
+            <>
+              <GridItem md={3} id="migration-type-label">
+                Hooks
               </GridItem>
-              <GridItem span={2} className="migration-hooks-align" />
-              <GridItem span={5} id="migration-plan-hooks-steps-label">
-                <label className="pf-c-form__label">Migration step</label>
-                {plan.spec.vms[0].hooks?.map((hook) => hook.step)}
+              <GridItem md={9} id="review-plan-hooks" aria-labelledby="migration-hooks-label">
+                <div>
+                  <Grid>
+                    <GridItem span={5} id="migration-plan-hooks-definition-label">
+                      <Text className={text.fontWeightBold}>Definition</Text>
+                      {plan.spec.vms[0].hooks?.map((hook) => (
+                        <Text key={Math.random()}>
```

Comment:
We shouldn't use a random number as a key, the key should be unique to the array element. Definitely overkill here performance-wise, but it's a good habit anyway. I'd use `key={hook.step}` maybe, since we'll currently only ever have two.